frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -55,6 +61,11 @@ export default {
     title() {
       return this.scheduleEntry.activity().title
     },
+    progressLabel() {
+      return this.scheduleEntry.activity().progressLabel !== null
+        ? this.scheduleEntry.activity().progressLabel().title
+        : null
```

Comment:
```suggestion
      return this.scheduleEntry.activity().progressLabel?.().title
```
See [optional chaining syntax for function calls](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls)